pkgs/by-name/bp/bpftrace/package.nix

```diff
@@ -48,6 +51,7 @@ stdenv.mkDerivation rec {
   # Pull BPF scripts into $PATH (next to their bcc program equivalents), but do
   # not move them to keep `${pkgs.bpftrace}/share/bpftrace/tools/...` working.
   postInstall = ''
+    wrapProgram $out/bin/bpftrace --add-flags "-I ${glibc.dev}/include"
```

Comment:
Not a 100% sure if this is the right thing to do in the presence of cross-compilation. I assume there is some way to get the correct include path from `stdenv` somehow and set it up. but IDK how.